Bks.DataAccess.Mongo.Test/MongoConnectorTest.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using FakeItEasy;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MongoDB.Driver;
+
+namespace Bks.DataAccess.Mongo.Test
+{
+    [TestClass]
+    public class MongoConnectorTest
+    {
+        private const string DbName = "MY_DB_NAME";
+        private const string DbPrefix = "MY_DB_PREFIX";
+        private const string DbCollectionName = "MY_COLLECTION_NAME";
+
+        private readonly ILogger<MongoConnector> mockLogger = 
```

Comment:
Use  Init method with [TestInitialize] attribute set all test scope variables (mockLogger , classMaps, options).
This method runs before each test